std_msgs/Header header
ts800_ros/SubmapEntry[] submap
